numpy/core/src/_simd/_simd_data.inc.src

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
This file isn't recognized as `c` syntax - so will need an entry in `.gitattributes`

Reply:
done

Reply:
should I move the highlight commit into a separate pr?

Reply:
no need, it is fine here.